docs/getting-started/benchmark.md

```diff
@@ -0,0 +1,94 @@
+---
+weight: -6
+---
+
+# Run a Benchmark
+
+After [installing GuideLLM](install.md) and [starting a server](server.md), you're ready to run benchmarks to evaluate your LLM deployment's performance.
+
+Running a GuideLLM benchmark is straightforward. The basic command structure is:
+
+```bash
+guidellm benchmark --target <server-url> [options]
+```
+
+### Basic Example
+
+To run a benchmark against your local vLLM server with default settings:
```

Comment:
It may make sense to reference the max-seconds parameter below, since this is a very time consuming default.